pyblish_qml/models.py

```diff
@@ -247,7 +246,7 @@ def data(self, index, role=QtCore.Qt.DisplayRole):
             except Exception:
                 pass
 
-        return QtCore.QVariant()
+        return Item()
```

Comment:
A little unsure about this one, since `PySide2` does not have `QVariant`, I replace it with `Item` because the `self.items` was containing objects generated from `Item()`.

But since it did not cause any error when I deliberately raising Exception in that try-except scope for testing, so I think it's legit.

Reply:
Hm, did you try returning `None` instead? Or even `"QVariant"`? QML would interpret QVariant as an invalid item, but may interpret `Item()` as a valid one, when it isn't?

Reply:
> did you try returning None instead? Or even "QVariant"? 

I haven't ! And I just did, looks like they all works without error, just displaying stuff incorrectly.

![image](https://user-images.githubusercontent.com/3357009/73636036-cb227980-469f-11ea-8499-c1d604d955d6.png)

I'll change it to `"QVariant"` for consistency.

Reply:
Changed in commit 14b58f9.